rootfs/Dockerfile

```diff
@@ -9,28 +9,22 @@ RUN adduser --system \
 
 COPY . /app
 
-RUN apt-get update \
-	&& apt-get install -y \
-		gcc \
-		git \
-		libffi-dev \
-		libpq5 \
-		libpq-dev \
-		python3 \
-		python3-dev \
-		sudo \
-	&& ln -s /usr/bin/python3 /usr/bin/python \
-	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 \
-	&& mkdir -p /configs && chown -R deis:deis /configs \
-	&& pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt \
-	&& apt-get remove -y --auto-remove --purge \
-		gcc \
-		git \
-		libffi-dev \
-		libpq-dev \
-		python3-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev'; \
+	# disable source repos (speeds up apt-get update)
+	sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
```

Comment:
If a new docker-base image does this already (i.e. after https://github.com/deis/docker-base/pull/2 is merged, I assume the `FROM` would be updated), wouldn't this line be redundant unless additional sources had been added (which in this case, they have not)?